### PR TITLE
HOTFIX: Psp 3983

### PR DESCRIPTION
**Pull request by devinleighsmith**
pims production deployment git action(s)

### CODE REVIEW

File: .github/workflows/ci-cd-pims-master.yml [hunk 1]
```diff
@@ -81,6 +81,28 @@ jobs:
         run: |
           ./openshift/4.0/player.sh build api -apply
 
+  tag-images:
```
Comment (devinleighsmith) [inline .github/workflows/ci-cd-pims-master.yml:84]: uat hasn't been tagging its images, not a big deal, but it should be doing this going forward.

File: .github/workflows/deploy-prod-end.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+name: Hide prod maintenance page after deployment.
```
Comment (devinleighsmith) [inline .github/workflows/deploy-prod-end.yml:1]: The idea behind splitting this up is that there are manual steps involved in prod deployments, so it doesn't make sense to burn through all of the steps and then take down the maintenance page and post a completion notification as that would likely finish before the manual steps are completed.

Instead, the deployer would run the prod deploy start script, wait for it to finish, then run any manual steps, then run the prod deploy end script.

File: .github/workflows/depoy-prod-start.yml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    steps:
+      - name: Checkout Source Code
+        uses: actions/checkout@v2
+        with:
+          ref: master
+          fetch-depth: 0
+      - name: Display the maintenance page instead of the app
+        run: |
+          ./maintenance/maintenance.sh prod on
+      - name: "Get Previous tag"
```
Comment (devinleighsmith) [inline .github/workflows/depoy-prod-start.yml:58]: this mimics the get previous tag functionality from the pipeline.

File: maintenance/maintenance.sh [hunk 4]
```diff
@@ -58,18 +58,6 @@ then
   exit
 fi
 
-
```
Comment (devinleighsmith) [inline maintenance/maintenance.sh:61]: This check doesn't work for our use case - ie. a service account with access to only the tools project but with rbac permissions that allow interaction with resources in other projects (ie. dev).

**Review by FuriousLlama — APPROVED** [review]
If we are deploying on OS, why is it necessary for it to have downtime?

**Comment by devinleighsmith** [comment]
@FuriousLlama for now we have manual steps for every prod release that must be completed before the environment can be made available.

**Pull request merged**